backend/tests/conftest.py

```diff
@@ -1,18 +1,33 @@
+from datetime import datetime
+
 import pytest
 from fastapi.testclient import TestClient
+from sqlalchemy.orm import scoped_session
 
-from backend.constants import API_VERSION_PREFIX
+from backend.database import Session
+from backend.database.models import Project, User
 from src.backend.entrypoint import app
 
 
 @pytest.fixture()
-def test_client():
+def mock_user_id():
+    new_user = User(created_on=datetime.utcnow(), projects=[])
+    with scoped_session(Session)() as session:
```

Comment:
Can you elaborate on the need for `scoped_session`? From [the doc](https://docs.sqlalchemy.org/en/20/orm/contextual.html#unitofwork-contextual) I fail to see why we'd need it.
Wouldn't a transaction inside the session be sufficient?

I see that this created User will never be removed from the DB.
I suggested previously to look at [zimfarm's garbage collector](https://github.com/openzim/zimfarm/blob/main/dispatcher/backend/src/tests/integration/conftest.py#L84).

I think the approach is better: you declare what you created and it takes care of deleting it.

Reply:
The the reason why I use `scoped_session` is that I found if I don't use it when I call the `session.begin()` it will complain the thread error. Let me try again.

Reply:
I know why. I added an extra `commit` by mistake.